### PR TITLE
feat: refactored pluginInfo

### PR DESCRIPTION
**Pull request by devteni**

### CODE REVIEW

File: controllers/infoController.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+<<<<<<< HEAD
+=======
```
Comment (Adeniyii) [inline controllers/infoController.js:2]: Teni, you forgot to remove this stuff from your code. Please refactor.

File: routes/goalRoutes.js [hunk 2]
```diff
@@ -2,14 +2,28 @@
 const { Router } = require('express');
 const { getSingleGoal, getAllGoals, createGoal, updateSingleGoalById, getArchivedGoals, deleteGoal } = require('../controllers/goalController');
 
+
 const router = Router();
 
+<<<<<<< HEAD
```
Comment (Adeniyii) [inline routes/goalRoutes.js:8]: In this file too

File: controllers/infoController.js (hunk 1 above)
Comment (devteni) [reply controllers/infoController.js:2]: Okay bro

File: routes/goalRoutes.js (hunk 2 above)
Comment (devteni) [reply routes/goalRoutes.js:8]: Okay

**Pull request merged**